resources/js/components/Comment.vue

```diff
@@ -14,12 +14,15 @@
             {{ date }}
         </div>
 
-        <div class="mt-2" v-html="commentString"></div>
+        <div class="mt-2">
+            <read-more more-str=">>>" :text="commentString" link="#" less-str="<<<" :max-chars="150"></read-more>
```

Comment:
@v-bubar isn't there still the limit of 150 chars? 
https://youtrack.hq.kosmoskosmos.systems/issue/FEW-149

Reply:
@kosmonowt Yes. Then comes read more the "read more" button.